.travis.yml

```diff
@@ -6,8 +6,12 @@ jdk:
 
 android:
   components:
-    - build-tools-21.1.2
-    - android-21
+    - build-tools-22.0.1
+    - android-22
+    - extra-android-support
+    - extra-google-google_play_services
```

Comment:
Ah, I see. I played with different combinations of this for awhile trying to get the travis build to work, and wasn't 100% sure what was needed. Sorry!